aten/src/ATen/native/native_functions.yaml

```diff
@@ -6140,14 +6140,14 @@
     CUDA: legacy::cuda::_th_eig
 
 - func: svd.U(Tensor self, bool some=True, bool compute_uv=True, *, Tensor(a!) U, Tensor(b!) S, Tensor(c!) V) -> (Tensor(a!) U, Tensor(b!) S, Tensor(c!) V)
-  dispatch:
-    DefaultBackend: svd_out
+  # dispatch:
```

Comment:
Check out the new dispatch rules in the native/README. If writing an op composed of existing ops the dispatch keyword is Math:. If implementing a custom kernel then the dispatch keyword should probably be CPU, CUDA:.

Reply:
I followed your suggestions in commit 9bcf00b359add64d8a5f3ce003196ab90fe1df6e.
However, it is not clear to me the differences between the various kind of dispatches. In particular:

* what is the difference between using `dispatch: DefaultBackend` and leave dispatch empty? It looks to me that in both cases you end up calling the same function for all backends, don't you? Commit c7bfc0780a5b3aa279687feddf334ba40028131d seems to suggest that these two cases are treated differently by autograd, but I fail to see what is the underlying reasoning.

* in this particular case, what is the advantage of using `dispatch: Math` over leaving dispatch empty? In both cases the tests seem to be passing (although at the moment of writing the tests have not been completed yet).

Reply:
@ailzhang, maybe an opportunity to clarify the README?